### PR TITLE
Functionality added for GCS multipart upload policy.

### PR DESCRIPTION
**Pull request by neerajsinghal05**
Functionality added for GCS multipart upload policy.

### CODE REVIEW

**Comment by samdgupi** [comment]
Please add some details and jira explaining why this change is needed

File: src/main/java/io/cdap/e2e/utils/StorageClient.java [hunk 1]
```diff
@@ -93,4 +98,14 @@ public static Blob uploadObject(String bucketName, String objectName, String fil
       Files.readAllBytes(Paths.get(StorageClient.class.getResource("/" + filePath).toURI())));
   }
 
+  public static Bucket createBucketwithLifeCycle(String bucketName, int age) throws IOException {
```
Comment (samdgupi) [inline src/main/java/io/cdap/e2e/utils/StorageClient.java]: where is this getting called from ?

Comment (samdgupi) [inline src/main/java/io/cdap/e2e/utils/StorageClient.java]: please add a java doc. It is unclear what timeunit age is refering to

Comment (neerajsinghal05) [reply src/main/java/io/cdap/e2e/utils/StorageClient.java]: This is newly created method , which will be called form framework to GCP to enable the lifecycle rule in the bucket with Age.

Comment (neerajsinghal05) [reply src/main/java/io/cdap/e2e/utils/StorageClient.java]: Java doc has added for the method mentioned above.

Comment (samdgupi) [reply src/main/java/io/cdap/e2e/utils/StorageClient.java]: Please change the variable name to `ageInDays`

**Review by samdgupi — APPROVED** [review]

File: src/main/java/io/cdap/e2e/utils/StorageClient.java (hunk 1 above)
Comment (neerajsinghal05) [reply src/main/java/io/cdap/e2e/utils/StorageClient.java]: suggested changes has been done.

**Pull request merged**